composer.json

```diff
@@ -5,8 +5,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.1",
-        "sylius/sylius": "~1.2.0",
-        "symfony/symfony": "^3.4|^4.1"
+        "sylius/sylius": "~1.3.0"
```

Comment:
```suggestion
        "sylius/sylius": "^1.2.0"
```

Reply:
Wouldn't it be better to create a new tag for this PR and keep this change? The readme was updated to fit for Sylius v1.3.

Reply:
Yes